.github/workflows/ci.yaml

```diff
@@ -80,6 +80,9 @@ jobs:
           cargo insta test --all --delete-unreferenced-snapshots
           git diff --exit-code
       - run: cargo test --package ruff_cli --test black_compatibility_test -- --ignored
+      # Check for broken links in the documentation.
+      # Setting RUSTDOCFLAGS because `cargo doc --check` isn't yet implemented (https://github.com/rust-lang/cargo/issues/10025).
+      - run: RUSTDOCFLAGS="-D warnings" cargo doc --all --no-deps
```

Comment:
Any reason to make this its own job? (Is this pretty fast?)

Reply:
In the CI for this PR it just took 38s. I mainly put it in the `cargo test` job because `cargo doc` also needs to compile the code and after `cargo test` the code already is compiled.